src/main/java/Duke.java

```diff
@@ -1,10 +1,440 @@
+    }
+
+    /**
+     * Show the current version's functionality of the bot
+     */
+    private static void printVersionDescription() {
+        //Uses list to store all the version information
+        List<String> versionDescriptions = new ArrayList<String>();
+        int maxDescriptionsLength = 0;
+        versionDescriptions.add("* " + CURR_VERSION);
+        versionDescriptions.add("* To know more about me, you can view my profile by typing the command \"view -p\"");
+        versionDescriptions.add("* For now I am a note bot that can help you note down any tasks and create a to-do list for you :)");
+        versionDescriptions.add("* In addition, you can mark any task in the to-do list as done!");
+        versionDescriptions.add("* You can type \"todo\" or \"deadline\" or \"event\" to create a task and I will help you save it automatically!");
+        versionDescriptions.add("* You can type \"list\" or \"ls\" to list all the tasks that are waiting to do");
+        versionDescriptions.add("* You can type \"done i\" where i is the index of the task to mark the specific task as done");
+        versionDescriptions.add("* You can type \"exit\" or \"bye\" to stop me and exit the program");
+        //Finds the length of the longest description to align all '*' displayed
+        for (String str : versionDescriptions) {
+            if (str.length() > maxDescriptionsLength) {
+                maxDescriptionsLength = str.length();
+            }
+        }
+        //Draws the frame for the version description
+        for (int i = 0; i < maxDescriptionsLength + 2; i++) {
+            System.out.print("-");
+        }
+        System.out.println();
+        for (String str : versionDescriptions) {
+            //Uses format string to print the '*' at the correct position after each sentence is completed
+            System.out.printf("%1$-" + (maxDescriptionsLength + 1) + "s", str);
+            System.out.println("*");
+        }
+        for (int i = 0; i < maxDescriptionsLength + 2; i++) {
+            System.out.print("-");
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //Prints all the welcome screens
+        printLogo();
+        printWelcomeMessage();
+        printVersionDescription();
+        System.out.println("\nLet's start:");
+        Task[] tasks = new Task[MAX_TASKS];
+        while (true) {
+            readCommand(tasks);
+        }
+    }
+
+    /**
+     * Reads the input command entered by the user and handles each command
+     *
+     * @param tasks The array to store all the tasks required
+     */
+    private static void readCommand(Task[] tasks) {
```

Comment:
Possible to use switch statements to handle the reading of commands

Reply:
Hmm actually I do consider using switch statements. However, since I also check if the user input contains only uppercase, lowercase or mixed, it makes me feel difficult in changing the if-else statements to switch case. Do you have any suggestion? Anyway, thanks for letting me know!